graphql/schema.graphql

```diff
@@ -608,18 +608,16 @@ input WorkstationSetupCommandInput {
 
 input PatchTriggerAliasInput {
   alias: String!
-  childProjectId: String!
   childProjectIdentifier: String!
```

Comment:
I wasn't able to successfully call the save mutation in a GraphQL playground; is that perhaps because because we're never actually setting the `childProjectId` based on the identifier? I don't see anywhere where that occurs. Let me know if you were able to get this working though

Reply:
Good call to test -- there was an issue with not handling the non-deprecated fields in the service functions. This should work as expected now.